chalice/awsclient.py

```diff
@@ -138,6 +138,17 @@ def get_rest_api_id(self, name):
                 return api['id']
         return None
 
+    def rest_api_exists(self, rest_api_id):
+        # type: (str) -> bool
+        """Check if an an API Gateway REST API exists."""
+        try:
+            self._client('apigateway').get_rest_api(restApiId=rest_api_id)
+            return True
+        except botocore.exceptions.ClientError as e:
```

Comment:
Yeah I'll get these updated, there's a few places I'm catching `ClientError` that can be updated.  I'll just need to bump the min botocore version to 1.5.0.